docs/guide/testing.md

```diff
@@ -23,9 +23,16 @@ Your test code will help you find bugs early, and alert you if you accidentally
 
 ## Testing frameworks for Textual
 
-Textual doesn't require any particular test framework.
-You can use any test framework you are familiar with, but we will be using [pytest](https://docs.pytest.org/) in this chapter.
+Textual is an async framework powered by Python's [asyncio](https://docs.python.org/3/library/asyncio.html) library.
+While Textual doesn't require a particular test framework, it must provide support for asyncio testing.
 
+You can use any test framework you are familiar with, but we will be using [pytest](https://docs.pytest.org/)
+along with the [pytest-asyncio](https://pytest-asyncio.readthedocs.io/) plugin in this chapter.
+
+By default, the `pytest-asyncio` plugin requires each async test to be decorated with `@pytest.mark.asyncio`.
+You can avoid having to add this marker to every async test
+by setting `asyncio_mode = auto` in your pytest configuration
+or by running pytest with the `--asyncio-mode=auto` option.
```

Comment:
Is this definitely accurate these days? I thought it was no longer required 🤔 I'm pretty sure I don't do this in Posting and the tests run fine.

Reply:
This was my understanding from the errors trying to follow the example in the Testing guide and reading the docs.